pshtt/build_pshtt.sh

```diff
@@ -23,19 +23,19 @@ source $VENV_DIR/bin/activate
 set -o nounset
 
 ###
-# Update pip and setuptools
+# Update pip, setuptools, and wheel
 ###
-pip install --upgrade pip setuptools
+pip install --upgrade pip setuptools wheel
 
 ##
 # Install pshtt
 ##
-pip install --upgrade pshtt==0.6.6
+pip install --upgrade pshtt==0.6.9
 
 ###
 # Install domain-scan
 ###
-mkdir domain-scan
+[ -d domain-scan ] || mkdir domain-scan
```

Comment:
Not to hold up progress but I'm confused on how this directory can exist before this command based on the Dockerfile and this script.

Reply:
When I ran `docker-compose up` I can into this issue (possibly because I had already run that command) and that caused my build to fail.  I threw in that code to avoid this (minor) issue in the future.

Reply:
Well it doesn't hurt anything so I'll just resolve this conversation. Just confusing behavior to hear about.

Reply:
Ahh right. I forgot it's reusing the container + the way these are building the lambda dependencies is... not great 😅.